lib/paths.cf

```diff
@@ -570,6 +586,31 @@ bundle common paths
 
       "path[shadow]"       string => "/etc/security/passwd";
 
+    termux::
+      # some adjustments for the basics
+      "path[tar]"           string => "/usr/bin/tar";
+      "path[true]"          string => "/usr/bin/true";
+      "path[false]"         string => "/usr/bin/false";
+      "path[sysctl]"        string => "/usr/bin/sysctl";
+
+      # now, mangle the values by prepending the TERMUX_PREFIX
+      "files_path" string => "/data/data/com.termux/files";
+      "etc_path" string => "$(files_path)/usr/etc";
```

Comment:
@nickanderson this is what I came up with after some experimentation. Too much? Too arbitrary that I chose bin, etc, var, tmp? I figured at least etc is super useful for "configuration" :) So I could limit it to that and bin maybe (used `$(paths.bin_path)` for python in packages.cf above.